.github/workflows/release-subsquid-processor.yml

```diff
@@ -0,0 +1,64 @@
+name: "Release Subsquid Production Image"
+env:
+  SUBSQUID_IMAGE_RELESE_TAGS: ${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:latest,${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:{{ github.event.pull_request.head.sha }},v0.0.1
+on:
+  push:
+    tags:
+      - 'subsquid-processor-v[0-9]+\.[0-9]+\.[0-9]+'  # e.g. subsquid-processor-v0.0.1
+
+jobs:
+  test-subsquid:
+    name: Test Subquid
+    runs-on:
+      - self-hosted
+      - linux
+      - x64
+      - sre
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+          persist-credentials: false
+      - name: Setup jest
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm ci
+        working-directory: subsquid
+      - name: Run tests
+        run: npm run test
+        working-directory: subsquid
+
+  subsquid-processor:
```

Comment:
I don't think this needs to be a separate job, can be the final step in the test portion?

Reply:
agree, it can be a step; will add that change